iFixit/react-commerce
# Upgrade to strapi v4

This PR closes #147 and #157 

## Changelog

> ⚠️ Disclaimer for reviewers: Almost all changes under `/backend` folder have been auto generated by replacing the old project with the new one. I've been working mostly on:
> - `/backend/src/plugins/seed` directory
> - docker files

- Upgrades Strapi to v4
- Refactor Strapi seed plugin according to v4 API changes
- Updates Dockerfile
- Makes frontend use [Govinor](https://govinor.com) strapi branch deployment.

> **Govinor** is a custom build deployment platform for Strapi (but it could in practice deploy any docker compose file). This will give us separate branch preview urls for Strapi. You can read more [here](https://github.com/iFixit/govinor). 

## QA

1. Visit the Vercel PR preview
2. Verify that the website works like in the main branch (https://react-commerce.vercel.app/)
3. Verify that an instance of Strapi v4 is running at https://upgrade-to-strapi-v4.govinor.com/admin
4. Try to change some data on Strapi and verify that the site updates accordingly
